android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -205,6 +205,11 @@ public Object getNodeValue(int nodeID) {
    * node is not present we try to return a "no-op" node that allows for "set" calls and always
    * returns 0 as a value.
    */
+
```

Comment:
remove empty line. I think the above comment should be moved down to be next to `findNodeById` method